gitpod-io/gitpod
# Replace YouTube with Mux for Gitpod demo video

## Description

Uses Mux for the demo video because of some Enterprise networking restrictions of YouTube. We already use Mux in Flex, so this was a pretty easy migration.

## Related Issue(s)

Fixes CLC-974
Fixes CLC-975

## How to test

1. Go to https://ft-muxier-videos.preview.gitpod-dev.com/workspaces
2. Observe the welcome video can still be played

/hold
